src/__test__/__snapshots__/HomePage.test.js.snap

```diff
@@ -0,0 +1,140 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/205280619-e0bf44eb-f58d-4170-a906-3e795636bd8a.PNG)

- [ ] Its great that you have tested your homepage component using the snapshots and mocked the api 👍 but you didn't test your redux which is a requirement for the testing of your application which you can see in the image above ⬆️ Kindly test your redux 👍

**Hint: You can see this [testing file](https://github.com/HAMMAS-SALEEM/CovidMonitor/blob/dev/src/testing/project.test.js) to get an idea what i'm talking about 😊**

Reply:
Hi @HAMMAS-SALEEM, thanks for your review and the hint given It was helpful to understand the requirements.